x-pack/plugins/lens/public/embeddable/embeddable_info_badges.tsx

```diff
@@ -66,11 +66,11 @@ export const EmbeddableFeatureBadge = ({ messages }: { messages: UserMessage[] }
               color: transparent;
               font-size: ${xsFontSize};
               height: ${euiTheme.size.l} !important;
-              .euiButtonEmpty__content {
-                padding: 0 ${euiTheme.size.xs};
+              .euiButtonEmpty {
+                padding-inline: ${euiTheme.size.xs};
               }
-              .euiButtonEmpty__text {
-                margin-inline-start: ${euiTheme.size.xs};
+              .euiButtonEmpty__content {
+                gap: ${euiTheme.size.xs};
```

Comment:
This changed caused a slight shift in the alignment of the icon. Do you know how we could correct this given the new makeup of the empty button?

Before: 
<img width="69" alt="Screenshot 2023-07-03 at 3 28 33 PM" src="https://github.com/elastic/kibana/assets/315764/1a9b2b5f-eba7-4320-8a78-df38fdfc2bf3">

After:
<img width="79" alt="Screenshot 2023-07-03 at 3 28 27 PM" src="https://github.com/elastic/kibana/assets/315764/d0ecea50-1e3d-470e-94d3-198f16bb1e2a">

Reply:
@drewdaemon you can use the `flush = 'left'` to adjust the alignment of the empty button